packages/playground/src/layouts/PipelineLayout.tsx

```diff
@@ -7,8 +7,8 @@ const PipelineLayout: React.FC = () => {
   const { pipelineId } = useParams<{ pipelineId: string }>()
 
   return (
-    <div>
-      <div style={{ display: 'flex', gap: '10px' }}>
+    <div className="h-[calc(100vh-100px)]">
+      <div className="flex p-2 gap-2.5 h-[44px] items-center">
```

Comment:
why do we need a fixed height and `items-center` here? adding the correct vertical padding will fix both height and alignment.

Reply:
fix all of them to work properly height with padding is 44px so I removed the 44px